pool.go

```diff
@@ -402,7 +402,7 @@ func (p *Pool) GetCtx(ctx context.Context, query string, bindings, rebindings ma
 }
 
 // OpenCursorCtx initiates a query on the database, returning a cursor to iterate over the results
-func (p *Pool) OpenCursorCtx(ctx context.Context, query string, bindings, rebindings map[string]string) (cursor Cursor, err error) {
```

Comment:
Should consider having a new function to create the cursor when used in the cursor.Read way, rather than confusing OpenCursorCtx and ReadCursorCtx where the open is used but the read is not. Possibly `Create StreamCursor` or similar